content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -14,201 +19,184 @@ the *bill control* policy chosen in the *Purchase* app settings.
-:guilabel:`Invoicing` section, and select :guilabel:`Ordered quantities` under :guilabel:`Bill
-Control`. Then, click :guilabel:`Save` to save changes.
+Ordered quantities
+------------------
 
-Next, go to the :menuselection:`Purchase` app, and click :guilabel:`Create` to create a new request
-for quotation (RFQ). Doing so reveals a blank :abbr:`RFQ (request for quotation)` detail form.
+To create and manage vendor bills for receipts with the bill control policy set to *ordered
+quantities*, first navigate to the :menuselection:`Purchase` app, and click :guilabel:`New`.
 
-On the blank detail form, add a vendor to the :abbr:`RFQ (request for quotation)` in the
-:guilabel:`Vendor` field, and add products to the :guilabel:`Product` lines by clicking
-:guilabel:`Add a line`.
+Doing so opens a new Request for Quotation (RfQ) form. On the blank |RfQ| form, add a
+:guilabel:`Vendor`, and click :guilabel:`Add a line` under the :guilabel:`Product` tab to add
+products to the order.
 
-Then, confirm the :abbr:`RFQ (request for quotation)` by clicking the :guilabel:`Confirm Order`
-button above the detail form. Doing so turns the :abbr:`RFQ (request for quotation)` into a purchase
-order.
+Once ready, click :guilabel:`Confirm Order` to confirm the |RfQ| into a |PO|.
 
-Then, click the :guilabel:`Create Bill` button to create a vendor bill for the purchase order.
+Then, click :guilabel:`Create Bill` to create a vendor bill. This opens a :guilabel:`Draft Bill`
+form. From here, add a billing date in the :guilabel:`Bill Date` field.
 
-Clicking the :guilabel:`Create Bill` button reveals the :guilabel:`Draft Bill` page for the purchase
-order.
-
-On the :guilabel:`Draft Bill`, click the :guilabel:`Edit` button to modify the bill, and add a bill
-date in the :guilabel:`Bill Date` field. If needed, add additional products to the
-:guilabel:`Product` lines by clicking :guilabel:`Add a line` in the :guilabel:`Invoice Lines` tab.
-
-Next, confirm the bill by clicking the :guilabel:`Confirm` button on the :guilabel:`Draft Bill`
-page.
+Once ready, confirm the bill by clicking :guilabel:`Confirm` on the :guilabel:`Draft Bill` page.
 
 .. tip::
    Since the bill control policy is set to *ordered quantities*, the draft bill can be confirmed as
    soon as it is created, before any products have been received.
 
-On the new :guilabel:`Vendor Bill`, add a :guilabel:`Bill Reference` number, which can be used to
-match the bill with additional documents (such as the :abbr:`PO (purchase order)`). Then, click
-:menuselection:`Confirm --> Register Payment`. Doing so causes a pop-up to appear, wherein a payment
-:guilabel:`Journal` can be chosen; a :guilabel:`Payment Method` selected; and a :guilabel:`Recipient
-Bank Account` can be selected from a drop-down menu.
+On the new :guilabel:`Vendor Bill`, add a :guilabel:`Bill Reference` number. This number can be used
+to match the bill with additional documents, such as the |PO|.
```

Comment:
maybe mention in a tip admonition block that you can access the PO using the Purchases smart button at the top?

![image](https://github.com/odoo/documentation/assets/23611711/c03d378a-a92c-46ed-8654-8e4efb1791e8)